packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-time-on-ward.component.tsx

```diff
@@ -11,12 +14,38 @@ const WardPatientTimeOnWard: React.FC<WardPatientTimeOnWardProps> = ({
   encounterAssigningToCurrentInpatientLocation,
 }) => {
   const { t } = useTranslation();
-  if (encounterAssigningToCurrentInpatientLocation) {
-    const timeOnWard = age(encounterAssigningToCurrentInpatientLocation.encounterDatetime);
-    return <div>{t('timeOnWard', 'Time on this ward: {{timeOnWard}}', { timeOnWard })}</div>;
-  } else {
-    return <></>;
+
+  if (!encounterAssigningToCurrentInpatientLocation?.encounterDatetime) {
+    return null;
+  }
+
+  const dur = dayjs.duration(dayjs().diff(encounterAssigningToCurrentInpatientLocation.encounterDatetime));
+  const days = Math.floor(dur.asDays());
+  const hours = dur.hours();
+  const minutes = dur.minutes();
+  const seconds = dur.seconds();
+
+  const parts = [];
+  // TODO: Fix translations to use the correct pluralization.
+  // Keys and strings extracted from translations below don't use the correct plural suffixes.
+  if (days > 0) {
+    parts.push(t('days', '{{count}} days', { count: days }));
```

Comment:
Looking at their [tutorial](https://youtu.be/SA_9i4TtxLQ?si=44WWfsUnsgn2tVF7&t=485), I think we're not supposed to have a default string.
```suggestion
    parts.push(t('days', { count: days }));
```
I tested locally and it seems to work (I did change the translation strings in `en.json` manually to have `"days_one": "{{count}} day"` ):
<img width="1710" alt="Screenshot 2025-05-07 at 1 01 43 PM" src="https://github.com/user-attachments/assets/e5e94e44-cfc1-49c5-9d4d-49a9bb40a4f3" />

Reply:
I haven't yet determined why we need to manually change the default value in `en.json` to get pluralization working correctly. As a last resort, we could add a comment block with hardcoded key and default value pairings that have the correct suffixes, but I'd prefer to avoid that approach. I was hoping someone might have encountered this before and has a straightforward solution.